docs/Explanation/Root-Verity.md

```diff
@@ -1,14 +1,109 @@
-
 # Root Verity
 
-<!--
-DELETE ME AFTER COMPLETING THE DOCUMENT!
----
-Task: https://dev.azure.com/mariner-org/polar/_workitems/edit/13118
-Title: Root Verity
-Type: Explanation
-Objective:
-
-Explain what root verity is, how it is implemented in Trident, what benefits and
-drawbacks it has. List all requirements and restrictions.
--->
+Root verity is a specific utilization of Verity, an integral part of the kernel
+that ensures that I/O for anything on the protected filesystem (in this case,
+root: `/`) is verified against a known good state. This is achieved by creating
+a hash tree of the root filesystem contents, which is then used to validate the
+integrity of the data being accessed.
+
+The hash tree is visualized in the
```

Comment:
Alot of this is common to user verity. Should we have one document that explains both?

Reply:
up to you.  are you asking but really saying "do this?"

Reply:
next round of doc restructuring maybe?